sdk/servicebus/azure-servicebus/azure/servicebus/aio/_base_handler_async.py

```diff
@@ -69,6 +93,17 @@ def __init__(
         self._auth_uri = None
         self._properties = create_properties(self._config.user_agent)
 
+    @classmethod
+    def _convert_connection_string_to_kwargs(cls, conn_str, **kwargs):
+        # pylint:disable=protected-access
+        return BaseHandlerSync._convert_connection_string_to_kwargs(conn_str, **kwargs)
```

Comment:
do we need to declare the method here? cuz it's only calling its parent's method.

Reply:
Not parent, it's the sync implementation.  Kinda begs a mixin, but the inheritence chain in handler is already a bit wacky.  Let me know what you think about my comment above, since that's the root of this whole workaround.

Reply:
aha, I see there is no inheritance.

Reply:
as above https://github.com/Azure/azure-sdk-for-python/issues/13683